### PR TITLE
Replaced "Slate" in Slate guides w/ "Staking API"

### PR DESCRIPTION
**Pull request by yash-sharma1**

### CODE REVIEW

File: docs/guides/staking-api-flows/avalanche/delegate/4_submit-signed-transaction.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Submit Signed Transaction for Broadcast
 ---
 
-After signing the transaction, you will `PUT /api/v1/flows/[:flow_id]/next` (as always!) and Slate will manage broadcasting the transaction to the NEAR network.
+After signing the transaction, you will `PUT /api/v1/flows/[:flow_id]/next` (as always!) and Staking API will manage broadcasting the transaction to the NEAR network.
```
Comment (pyr0gan) [inline docs/guides/staking-api-flows/avalanche/delegate/4_submit-signed-transaction.mdx:5]: Most usages of "Staking API" in a sentence will require the definite article, the word "the", in front of them.

**Review by pyr0gan — COMMENTED** [review]
Looks great apart from the single issue we discussed, having the definite article.

**Comment by pyr0gan** [comment]
@yash-sharma1 I fixed the images as mentioned in the Jira ticket - see commit https://github.com/figment-networks/figment-documentation/pull/88/commits/912f92df13ccc8b8679d435706781517848095f5 🚀

**Review by paulpomerleau — APPROVED** [review]
lgtm

**Pull request merged**